File: C#/Language/helloworld/helloworld.cs
using System;    /*using关键字用在程序中包含System命名空间*/

namespace HelloWorldApplication      /*一个namespace里包含一系列的类*/
{
	class HelloWorld
	{
		static void Main(string[] args)       //注意C#中Main函数必须大写，Main方式是所有C#程序的入口点
		{
			Console.WriteLine("Hello World!");    //WriteLine是一个定义在System命名空间中的Console类的一个方法
			Console.ReadKey();                    //防止程序从 Visual Studio .NET 启动时屏幕会快速运行并关闭。
		}
	}                                         //C#中类、结构体、枚举定义结尾不需要加分号结尾，而C++中则必须加
}

File: C++/Language/helloworld/compile/helloworld.cpp
/*Visual Studio安装目录VC\Tools下的一些常用工具：
 *1.编译器cl.exe
 *2.汇编器ml.exe/ml64.exe
 *3.衔接器link.exe
 *4.静态库打包工具lib.exe
 *5.用于Makefile文件构建和编译工具nmake.exe
 *6.查看dll或exe的依赖库工具dumpbin.exe
 */

#include <stdio.h>
#include <stdlib.h>
#include <string.h>

#define SIZE 16

char *str = (char*)malloc(SIZE);

int main(void)
{
	memset(str,0,SIZE);
	strcpy(str,"hello world");
	puts(str);
	
	return 0;
}